src/Viewer/components/Monaco/MonacoInstance.js

```diff
@@ -137,22 +131,16 @@ function MonacoInstance ({
                 ],
             },
         });
-        setLanguage("logLanguage");
-    }
+    };
 
 
     /**
      * Called when editor is finished mounting.
-     *
-     * @param {object} editor
-     * @param {object} monaco
      */
-    const handleEditorDidMount =(editor, monaco) => {
-        monacoRef.current = monaco;
-        editorRef.current = editor;
+    const handleEditorDidMount = () => {
```

Comment:
Consider renaming this to `_onMount` so we are consistent with the name in the component props.